base/operators.jl

```diff
@@ -132,14 +132,22 @@ isequal(x::AbstractFloat, y::Real         ) = (isnan(x) & isnan(y)) | signequal(
 """
     isless(x, y)
 
-Test whether `x` is less than `y`, according to a canonical total order. Values that are
-normally unordered, such as `NaN`, are ordered in an arbitrary but consistent fashion.
+Test whether `x` is less than `y`, according to a fixed total order.
+`isless` is not defined on any pair of values `(x, y)`. However, if it
+is defined, it satisfies the following:
+- If `isless(x, y) is defined, then so is `isless(y, x)` and `x == y`,
```

Comment:
>If `isless(x, y) is defined, then so is `isless(y, x)` and `x == y`,

If `isless(x, y) is defined, then so is `isless(y, x)` and `isequal(x,y)`,

Reply:
Thx, I fixed that.